jck/jtrunner/JavaTestRunner.java

```diff
@@ -374,19 +374,27 @@ public static boolean generateJTB() throws Exception {
 
 		if ( getJckVersionInt(jckVersionNo) >= 18 && (tests.contains("api/java_net") || tests.contains("api/java_util")) ) {
 			// Requires SHA1 enabling for jar signers in jdk-18+
-			secPropsFile = resultDir + File.separator + "security.properties";
-			System.out.println("Custom security properties to be stored in: " + secPropsFile);
 			String secPropsContents = "jdk.jar.disabledAlgorithms=MD2, MD5, RSA keySize < 1024, DSA keySize < 1024";
 			BufferedWriter bw = new BufferedWriter(new FileWriter(new File(secPropsFile)));
 			bw.write(secPropsContents);
 			bw.flush();
 			bw.close();
 		}
 		
+		if ( tests.contains("api/java_net") || tests.contains("api/java_util")) {
```

Comment:
This section is going to conflict with the if above line 375? I think this needs combining under a common if for (tests.contains("api/java_net") || tests.contains("api/java_util") ?

Reply:
> This section is going to conflict with the if above line 375

It would not create a conflict since I used `bw.append(secPropsContents); ` inside that if, So it would preserve whatever settings get written in security.properties file by the if block in line 375. 

I did it this way since the properties written in line 377 are slightly different than what I need to add for my fix. 

However, I do see you have a similar PR in which you changed the condition from 18+ to 11+ in your PR (https://github.com/adoptium/aqa-tests/pull/4059/files). 

I think we should have a single PR that capture all these changes. 

I will combine the two PRs and retest.

Reply:
thanks